docs: insert link to angular commit format (#132)
T-Systems-MMS/phonebook: docs/pages/development-guides/general-guide.md

1. Create a new Branch.
2. Make some changes and develop a cool new feature or fix a Bug.
3. In order to commit run `npm run commit` to run the interactive Commit utility.

> > We are using the angular [angular commit format reference](https://gist.github.com/brianclements/841ea7bffdb01346392c). All commits should be in this format because of automatic releases.

Reviewer: ```suggestion
> We are using the angular [angular commit format reference](https://gist.github.com/brianclements/841ea7bffdb01346392c). All commits should adhere to this format in order to trigger automatic releases.
```
